lib/GADS/Schema/Result/Report.pm

```diff
@@ -304,12 +322,30 @@ sub create_pdf
         fg_color  => '#ffffff',
     };
 
-    $pdf->table(
-        data         => $fields,
-        header_props => $hdr_props,
-        border_c     => '#007c88',
-        h_border_w   => 1,
-    );
+    foreach my $topic (keys %$grouped_data) {
+        next if $topic eq 'Other';
+        my $fields = [ [$topic, ''] ];
+        push( @{$fields}, [ $_->{name}, $_->{value} ] ) foreach (@{$grouped_data->{$topic}});
```

Comment:
As discussed, please combine this code with the similar code in `GADS::Role::Presentation::Record`, moving the overall code in `GADS::Record`